I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 131, in script call
    call a_The_Build_Up from _call_a_The_Build_Up
  File "game/chapters/a_The_Build_Up.rpy", line 20, in script call
    call check_100_posters from _call_check_100_posters
  File "game/utilities/check_100_posters.rpy", line 2, in script
    python:
  File "game/utilities/check_100_posters.rpy", line 5, in <module>
    from game.chapters.hacks import hack_100_posters
ImportError: No module named game.chapters.hacks

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 131, in script call
    call a_The_Build_Up from _call_a_The_Build_Up
  File "game/chapters/a_The_Build_Up.rpy", line 20, in script call
    call check_100_posters from _call_check_100_posters
  File "game/utilities/check_100_posters.rpy", line 2, in script
    python:
  File "/Users/princesssteffykins/Downloads/RENPY PROJECTS/renpy-7.2.2-sdk/renpy/ast.py", line 900, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/princesssteffykins/Downloads/RENPY PROJECTS/renpy-7.2.2-sdk/renpy/python.py", line 1930, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/utilities/check_100_posters.rpy", line 5, in <module>
    from game.chapters.hacks import hack_100_posters
ImportError: No module named game.chapters.hacks

Darwin-17.7.0-x86_64-i386-64bit
Ren'Py 7.2.2.491
Guaiya Means Love 1.0
Wed Jan 22 19:31:11 2020
